packages/s3-request-presigner/README.md

```diff
@@ -91,4 +91,46 @@ to `presigned` is not sufficient to make a request. You need to send the
 server-side encryption headers along with the url. These headers remain in the
 `presigned.headers`
 
+getSignedUrl Example:
+
+```javascript
+import { PutObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
+
+const s3Client = new S3Client({ region: 'us-east-1' });
+const command = new PutObjectCommand({
+    Bucket: bucket,
+    Key: key,
+});
+
+const presigned = getSignedUrl(this.inner, command, {
+      expiresIn: expiration,
+    });
```

Comment:
This is not a simple example, and is copied from some other application code.
The `this.inner` doesn't mean anything here.

Reply:
sorry about that, updated it to use the s3Client